stko/optimizers/collapser.py

```diff
@@ -228,60 +231,571 @@ def optimize(self, mol):
             shutil.rmtree(output_dir)
         os.mkdir(output_dir)
 
-        BB_ids = list(set([
+        bb_ids = list(set([
             i.get_building_block_id() for i in mol.get_atom_infos()
         ]))
-        BB_atom_ids = {i: [] for i in BB_ids}
+        bb_atom_ids = {i: [] for i in bb_ids}
```

Comment:
Just use a defaultdict from collections for this. Looks way more readable. It would just be `bb_atom_ids = defaultdict(list)`. Same thing, but way more readable.

Reply:
I like this!